src/agat/agat_sp_complement_annotations/config.vsh.yaml

```diff
@@ -0,0 +1,95 @@
+name: agat_sp_complement_annotations
+namespace: agat
+description: |
+    The script allows to complement a reference annotation with other annotations. 
+    
+      * A l1 feature from the addfile.gff that does not overlap a l1 feature from the reference annotation will be added. 
+      * A l1 feature from the addfile.gff without a CDS that overlaps a l1 feature with a CDS from the reference annotation will be added. 
+      * A l1 feature from the addfile.gff with a CDS that overlaps a l1 feature without a CDS from the reference annotation will be added. 
+      * A l1 feature from the addfile.gff with a CDS that overlaps a l1 feature with a CDS from the reference annotation will be added only if the CDSs don't overlap. 
+      * A l1 feature from the addfile.gff without a CDS that overlaps a l1 feature without a CDS from the reference annotation will be added only if none of the l3 features overlap. 
+      
+      ! It is sufficient that only one isoform is overlapping to prevent the whole gene (l1 feature) from the addfile.gff to be added in the output.
```

Comment:
```suggestion
      It is sufficient that only one isoform is overlapping to prevent the whole gene (l1 feature) from the addfile.gff to be added in the output.
```